torch/onnx/symbolic_opset11.py

```diff
@@ -928,13 +928,21 @@ def repeat_interleave(g, self, repeats, dim=None, output_size=None):
             output_sizes[idx], input_sizes[idx] = 0, -1
     perm_i[0], perm_i[dim] = perm_i[dim], perm_i[0]
 
-    # Cases when repeats is a single value tensor and dim has unknown input size
-    if (repeats_dim == 0 or (repeats_dim == 1 and repeats_sizes[0] == 1)) and output_sizes[dim] == 0:
-        if not sym_help._is_tensor(repeats):
-            repeats = g.op("Constant", value_t=torch.LongTensor(repeats))
+    # If input size is dynamic or repeats vector is dynamic
+    if output_sizes[dim] == 0 or (repeats_dim == 1 and repeats_sizes[0] is None):
```

Comment:
Use a boolean for `repeats_dim == 1 and repeats_sizes[0] is None`, because it is reused below.
The comment is if input size is dynamic, why the if condition is `output_sizes[dim] == 0` not `input_sizes`?